sonic-xcvrd/scripts/xcvrd

```diff
@@ -20,6 +20,7 @@ try:
     from sonic_py_common import daemon_base, device_info, logger
     from swsscommon import swsscommon
     from sonic_py_common import multi_asic
+    from . import xcvrd_utilities
```

Comment:
Add blank line above this to separate third-party imports from local imports per PEP8 standards.

Reply:
fixed